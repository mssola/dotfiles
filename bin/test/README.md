
# Lalala

A readme file

Copyright &copy; 2014-2015 Miquel Sabaté Solà, released under the MIT License.

