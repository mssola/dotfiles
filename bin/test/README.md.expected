
# Lalala

A readme file

Copyright &copy; 2014-2018 Miquel Sabaté Solà, released under the MIT License.

